int16[] blocks_over
int16 LEFT_ARM=0
int16 RIGHT_ARM=1
int16 num_arms
int16[2] gripper_at
bool[2] gripper_closed


